map/navbar.php

```diff
@@ -0,0 +1,45 @@
+<?php function navbar($suffixNavItems = null, $prefixNavItems = null, $additionalItems = null, $brandkLink = "index.php") { ?>
+    
+
+
+<nav class="navbar navbar-expand-md navbar-dark fixed-top" style="background-color: #9F0C35;">
+    <a class="navbar-brand" href="<?php echo $brandkLink ?>" active>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$brandkLink'.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)